src/platforms/javascript/guides/gatsby/index.mdx

```diff
@@ -37,8 +37,8 @@ Register the plugin in your Gatsby configuration file (typically `gatsby-config.
       },
     },
     // ...
-  ];
-}
+  ],
+};
```

Comment:
Hmm maybe we go ahead and make this closer to Gatsby docs -- https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/

```
module.exports = {
  plugins: [
    //plugins go here
  ],
}
```

Reply:
I'd rather we stay consistent with our code styling throughout docs.

Reply:
We declare this file to be `gatsby-config.js`. Does it actually work with Gatsby without the `module.exports` part? For the rest -- yeah I think we should have a `prettier` config in this repo so just apply that for consistency is okay.

Reply:
My mistake, I misunderstood the nature of your comment. Yeah we need to add `module.exports` as it's a javascript file.

Reply:
I also missed the `module.exports` hehe. Should be fixed now, including the onboarding wizard. LMK what you think.